src/components/Selectors/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from 'react';
+import { CheckBoxProps, SelectorSize } from '../Selectors.types';
+import { classNames } from '../../../shared/utilities';
+
+import styles from '../selectors.module.scss';
+
+export const CheckBox: FC<CheckBoxProps> = ({
+    ariaLabel,
+    checked = false,
+    color,
+    defaultChecked = false,
+    disabled = false,
+    inputType = 'checkbox',
+    id,
+    name,
+    size = SelectorSize.Medium,
+    value,
+    onClick,
+}) => {
+    const checkBoxDivClassNames: string = classNames([styles.selector]);
+
+    const checkBoxInputWrapperClassNames: string = classNames([
+        styles.selectorWrapper,
+    ]);
+
+    const checkBoxLabelClassNames: string = classNames([styles.selectorLabel]);
+
+    const checkBoxCheckClassNames: string = classNames([
+        styles.checkmark,
+        { [styles.selectorSmall]: size === SelectorSize.Small },
+        { [styles.selectorMedium]: size === SelectorSize.Medium },
+        { [styles.selectorLarge]: size === SelectorSize.Large },
+        { [styles.red]: color === 'red' },
+        { [styles.orange]: color === 'orange' },
+        { [styles.yellow]: color === 'yellow' },
+        { [styles.green]: color === 'green' },
+        { [styles.bluegreen]: color === 'bluegreen' },
+        { [styles.blue]: color === 'blue' },
+        { [styles.violet]: color === 'violet' },
+        { [styles.grey]: color === 'grey' },
```

Comment:
We should not indiviually need to handle colors like this

Reply:
I was following how the Pill component is using the OcThemeNames type to configure colors. Is there another way I can do this or that the theme provider can handle the colors?

Reply:
If you look at the pills in the octuple figma they are avaible in all theme colors. 
But checkox and radio need to follow the primary color.  
Lets have a call today to discuss on how we can handle this